package.json

```diff
@@ -9,6 +9,7 @@
   ],
   "author": "The Terasology Foundation",
   "dependencies": {
+    "@reach/router": "^1.3.4",
```

Comment:
Used for the higher-order component, but may not actually be needed. Need to double check...